src/utils/vscode/getPRsToPaintPerSHAs.ts

```diff
@@ -1,9 +1,12 @@
+import * as vscode from "vscode";
 import countOrganizationQueries from "../countOrganizationQueries";
 import getPRsPerSHAS from "../getPRsPerSHAS";
 import getIssue from "../github/getIssue";
 import getIssueComments from "../github/getIssueComments";
 import getRepoInfo from "./getRepoInfo";
 import { noLinesSelected, noSearchResults } from "./showErrors";
+import getSHAArray from "../getSHAArray";
+import getGitAPI from "../vscode/getGitAPI";
```

Comment:
This gets created on activation

Reply:
How would I do it another way? The only way to get the prs is in the file we're modifying itself. 

But, we only need to modify the list of SHAs we're getting (in this case basing it not on selection, but as a variable number of lines).